jarviscli/plugins/bulkresize.py

```diff
@@ -0,0 +1,70 @@
+import cv2
+import os
+import glob
+
+from plugin import plugin
+
+
+def bulk_resizer(input_path, output_path, desired_size=32,
+                 color=[0, 0, 0], rename=True):
+    img_no = 0
+
+    onlyfiles = os.listdir(input_path)
+    # print(onlyfiles)
+    for im_pth in onlyfiles:
+        try:
+            im = cv2.imread(im_pth)
```

Comment:
im_pth does at this point not contain the full path - but only the filename (e.g. jarvis.png).
=> This line will fail for all images not in working directory, you should prepend input_path.

Apropos, there is even in python stdlib: 
https://docs.python.org/3/library/os.path.html#os.path.join

Reply:
Added